diff --git a/dist/index.mjs b/dist/index.mjs
index 24a1e7c7d6493baebe3bacfd21597b2026cbd46c..23b865b1e081864e6358ee46128d0018abc96d44 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -272,7 +272,7 @@ const formatField = (field, fieldType)=>{
             return field;
     }
 };
-const formatRecord = (record, schema)=>Object.fromEntries(Object.entries(record).map(([key, value])=>[
+const formatRecord = (record, schema)=>Object.fromEntries(Object.entries(record ?? {}).map(([key, value])=>[
             key,
             formatField(value, schema[key])
         ]));